utils/BaseStore.js

```diff
@@ -1 +1,4 @@
-module.exports = require('dispatchr/utils/BaseStore');
+// @TODO deprecate
+//console.warn("require('fluxible/utils/createStore') is deprecated. Please use " +
```

Comment:
I learned the last time we added warnings that it can be annoying to suddenly see warnings without having done anything. I'd rather do it when the explicitly upgrade.